cmd/create/cluster/cmd.go

```diff
@@ -1967,7 +1967,7 @@ func run(cmd *cobra.Command, _ []string) {
 	privateSubnetsCount := 0
 
 	var availabilityZones []string
-	var subnets []*ec2.Subnet
+	var subnets []ec2types.Subnet
```

Comment:
Why did these change from a pointer?

Reply:
It was changed to simplify the code due to a lot of the functions in v2 switched to not use pointers and it minimises a lot of unecessary conversions, It was made from Ori's request while reviewing https://github.com/openshift/rosa/pull/1564